src/Qir/Tools/Executable/QirFullStateExecutable.cs

```diff
@@ -21,17 +21,15 @@ public class QirFullStateExecutable : QirExecutable
         public override string DriverFileExtension => "cpp";
 
         public override IList<string> LinkLibraries => new List<string> {
-                "Microsoft.Quantum.Qir.Runtime",
-                "Microsoft.Quantum.Qir.QSharp.Foundation",
-                "Microsoft.Quantum.Qir.QSharp.Core"
+                "Microsoft.Quantum.Simulator.Runtime"
```

Comment:
Nit, but this should likely use next-line brace style.

Reply:
Adjusted!